src/Microsoft.DotNet.Interactive/PackageRestoreContext.cs

```diff
@@ -25,6 +25,8 @@ public class PackageRestoreContext : IDisposable
 
         public PackageRestoreContext()
         {
+            // By default look in to the package source  "https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json"
+            AddRestoreSource("https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json");
```

Comment:
Let's also add our own feed for now: https://dotnet.myget.org/F/dotnet-try/api/v3/index.json